engine-api/src/methods/chain_id.rs

```diff
@@ -0,0 +1,41 @@
+use {
+    crate::jsonrpc::JsonRpcError,
+    moved::{genesis::config::CHAIN_ID, types::state::StateMessage},
+    tokio::sync::mpsc,
+};
+
+pub async fn execute(
+    _request: serde_json::Value,
+    _state_channel: mpsc::Sender<StateMessage>,
+) -> Result<serde_json::Value, JsonRpcError> {
+    Ok(serde_json::to_value(format!("0x{:x}", CHAIN_ID))
```

Comment:
This should be based on the genesis config in the state actor instead of returning a static response because it will be important to return different chain ids even for our own networks (devnet, testnet, mainnet, must all be different).

It would also be fine to pass the genesis config down to this method instead of asking the state actor for it since the genesis config is static for the whole execution of the chain.

Reply:
Getting it from the genesis config via state actor now. Chain id still comes out correct:
```json
"op_move_response": {
    "id": 1,
    "jsonrpc": "2.0",
    "result": "0x194"
}
```